src/woo_publications/publications/models.py

```diff
@@ -304,7 +295,11 @@ def draft(self) -> None:
     @transition(
         field=publicatiestatus,
         # type annotations are missing and pyright infers that only string is allowed
-        source=("", PublicationStatusOptions.concept),  # pyright:ignore[reportArgumentType]
+        source=(
+            "",
+            PublicationStatusOptions.concept,
+            PublicationStatusOptions.published,
```

Comment:
this one is not okay - it doesn't make semantic sense that you publish something that's already published. It's a one-time transaction.